source/text/Format.cpp

```diff
@@ -384,6 +384,14 @@ string Format::Number(double value)
 		return "???";
 	else if(std::isinf(value))
 		return value > 0. ? "infinity" : "-infinity";
+	else if(fabs(value) > 1000000000000000ll)
```

Comment:
```suggestion
	else if(fabs(value) > 1e15ll)
```
(if I counted the zeros correctly)

Reply:
I just copy-pasted it from Format::Credits, which also had it in this form.

Reply:
I still don't like it, but do as you prefer